cylc/flow/task_events_mgr.py

```diff
@@ -108,6 +109,44 @@ def log_task_job_activity(ctx, suite, point, name, submit_num=None):
         LOG.debug(ctx_str)
 
 
+class EventData(Enum):
```

Comment:
Enumeration of the event handler data fields. Not strictly necessary but humour me, I like enumerations. The plan was to auto-document from this enumeration in the config for further safety.

Sadly I can't get it to format nicely in the docs (it renders as `Event='event'` whereas we want `%(event)s`). Would be more work to fix this than we have time to do at the moment so I've dropped it for now.